recipe/meta.yaml

```
@@ -147,7 +154,7 @@ outputs:
         - gmp
         - zlib
         - libiconv
-        - cctools_{{ cross_target_platform }}
+        - cctools  # _{{ cross_target_platform }}
```

Necessary to keep in the recipe? Or would we rather

```suggestion
        - cctools 
```

---

Yes, I should replace those with the comment from the PR description.